### PR TITLE
Adding blank project

### PR DESCRIPTION
**Pull request by piotrpekala7**

### CODE REVIEW

File: src/app/components/projects/add-blank-project-dialog/add-blank-project-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        }
+
+    ngOnInit() {}
+
+    get form() {
+        return this.projectNameForm.controls;
+    }
+
+    onAddClick() : void{
+        if (!this.projectNameForm.invalid){
```
Comment (ziajka) [inline src/app/components/projects/add-blank-project-dialog/add-blank-project-dialog.component.ts]: reverse condition

File: src/app/components/projects/models/validator.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export class Validator {
+    static projectNameValidator(projectName) {
```
Comment (ziajka) [inline src/app/components/projects/models/validator.ts]: we don't like static

File: src/app/components/projects/add-blank-project-dialog/add-blank-project-dialog.component.ts (hunk 1 above)
Comment (piotrpekala7) [reply src/app/components/projects/add-blank-project-dialog/add-blank-project-dialog.component.ts]: I dunno what u mean

Comment (ziajka) [reply src/app/components/projects/add-blank-project-dialog/add-blank-project-dialog.component.ts]: if (this.projectNameForm.invalid){
return;
}

**Comment by ziajka** [comment]
I think we may also change to "create new project" and after successful creation user is redirected to newly created project.

**Comment by ziajka** [comment]
Thank you!

**Pull request merged**